source/common/upstream/cluster_manager_impl.cc

```diff
@@ -178,7 +178,14 @@ void ClusterManagerInitHelper::maybeFinishInitialize() {
 
 void ClusterManagerInitHelper::onStaticLoadComplete() {
   ASSERT(state_ == State::Loading);
-  state_ = State::WaitingForStaticInitialize;
+  // After initialization of primary clusters has completed, transition to
```

Comment:
Why was `state_` `WaitingForStaticInitialize` before but now for secondary?

Reply:
I've renamed states to better reflect cluster manager is initialization sequence.